metis/lib/models/data_block.rb

```diff
@@ -74,10 +73,9 @@ def temp_hash?
       md5_hash =~ TEMP_MATCH
     end
 
-    def compute_hash!
-      if has_data? && temp_hash?
+    def compute_hash!(&test_hook)
+      if temp_hash? and has_data?
```

Comment:
Any particular reason for `and` here?

Reply:
Nope, programming language fuzziness is all.